src/configuration.ts

```diff
@@ -172,6 +173,39 @@ export class ConfigurationManager implements vscode.Disposable
         return {arch: arch, os: os};
     }
 
+    private async getVcpkgPathFromEnv()
+    {
+        // let envVar = process.env.VCPKG_PATH;
```

Comment:
Finally, should we use it instead of `environmentVariableCollection` since `environmentVariableCollection` requires `vscode.ExtensionContext` which cannot be obtained from the test suite.

Related: https://github.com/microsoft/vscode/issues/152806

Reply:
I am not so sure about this.